.github/workflows/publish-main.yaml

```diff
@@ -0,0 +1,75 @@
+name: Publish to main 
+
+on:
+  push:
```

Comment:
This approach won't work, as the action will only be triggered when a new commit is added to the specified branch. We require something that will be triggered by execution of the "release" action and continue to retry until a condition is satisfied.

Calling the [schedule-job-action](https://github.com/marketplace/actions/schedule-job-action) from the "release" action should allow this.